domain/src/main/assets/wANbh4oOClga.json

```diff
@@ -19,7 +19,7 @@
         "objective": "Learn about fractions in detail.",
         "id": "umPkwp0L1M0-"
       },
-      "outline": "This is outline/summary for <b>What is a Fraction?</b>",
+      "outline": "This is outline/summary for <b>What is a Fraction?</b> Very long summary, it gets cut off after the fourth line. Very long summary, it gets cut off after the fourth line. Very long summary, it gets cut off after the fourth line. I need to fix it!",
```

Comment:
No changes required in this file. You can add in the description that to review this Pull Request, the reviewers need to modify this file in order to see the long description.